src/styles/_mixins.scss

```diff
@@ -22,58 +22,58 @@
 }
 
 @mixin input-default {
-  background-color: var(--KInputBackground, var(--white, color(white)));
-  box-shadow: inset 0 0 0 1px var(--KInputBorder, var(--grey-300)) !important;
+  background-color: var(--KInputBackground, var(--white, #ffffff));
```

Comment:
question: can we use `$kui-color-background` here?

Reply:
Same as with one of Adam's questions: tokens in standalone styles files (`mixins.scss, inputs.scss, etc.`) will be replaced in a separate ticket